detekt.properties

```diff
@@ -1,14 +1,20 @@
 category=External Analysers
 description=Provide detekt rules for Kotlin projects
 homepageUrl=https://github.com/detekt/sonar-kotlin
-archivedVersions=2.0.0
-publicVersions=2.1.0,2.2.0
+archivedVersions=2.0.0,2.2.0
+publicVersions=2.1.0,2.3.0
```

Comment:
Note: sonar-detekt 2.2.0 updated the sonar-api to 7.9.4 so 2.1.0 is still a public version for users of sonar < 7.9.4.

Reply:
thanks for the reminder! :)